sdk/python/tests/unit/test_registry_conflict.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from feast.entity import Entity
+
+
+@pytest.mark.integration
+def test_apply_first_entity(environment):
```

Comment:
If the Scopes have been changed i don't see a huge point to these tests - even if there's a race condition in the future it seems like the repro for that will need it's own test. Wdyt?

Reply:
yeah this is fair; I'll remove this in some future PR when I clean up test suite